src/main/java/algorithms/sorting/radixSort/RadixSort.java

```diff
@@ -16,9 +16,9 @@ public class RadixSort {
      * @return The value of the digit in the number at the given segment.
      */
     private static int getSegmentMasked(int num, int segment) {
-        // Bit masking here to extract each segment from the integer.
-        int mask = ((1 << NUM_BITS) - 1) << (segment * NUM_BITS);
-        return (num & mask) >> (segment * NUM_BITS);
+        // bit masking here to extract each segment from the integer.
```

Comment:
Actually I've been thinking - for students taking CS2040S, they have probably not touched CS2100 yet, so the concept of bit representation might be a bit complex for them. 

Would it be better to maybe insert more helpful links regarding how computers represent numbers (eg. float, integer).

Maybe we could add this link: https://cheever.domains.swarthmore.edu/Ref/BinaryMath/NumSys.html or somethign else to help clarify things.

Reply:
added